CHANGELOG.md

```diff
@@ -103,20 +101,20 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
   - Instead of calling `mempool()`, one should use `is_tx_in_pool`
   or `add_tx` methods.
 
-- `TestKitApi::get_err` method now returns `ApiError`, rather than
-  a deserialized object, as it is for `get`. For checking such results
+- `TestKitApi::get_err` method now returns `ApiError`, rather than a deserialized
+  object, as it is for `get`. For checking such results
   in tests you may want to use `assert_matches`.
 
 #### exonum-configuration
 
-- `majority_count: Option<u16>` configuration parameter is introduced.
-  Allows to increase the threshold amount of votes required to commit
-  a new configuration proposal. By default the number of votes is calculated
-  as 2/3 + 1 of total validators count. (#546)
+- `majority_count: Option<u16>` configuration parameter is introduced. Allows to
+  increase the threshold amount of votes required to commit a new configuration
+  proposal. By default the number of votes is calculated as 2/3 + 1 of total
+  validators count. (#546)
 
-- `validators-count` command-line parameter has been added. Now, when
-  generating config template using `generate-template` command, you must
-  specify the number of validators. (#586)
+- `validators-count` command-line parameter has been added. Now, when generating
```

Comment:
Maybe this feature description should be moved to the core section?

Reply:
This options is definitely part of the core configuration, but it only makes sense with the configuration service. I suppose it should be mentioned twice.